e2e/cypress/integration/ui/settings/project_mgmt.spec.ts

```diff
@@ -1,31 +1,33 @@
-import { itFlaky } from '../../../support/constants';
-interface Project {
-  id: string;
-  name: string;
-  type: string;
-}
+import { Project, Rule } from '../../../support/types';
 
 describe('project management', () => {
-  // we increase the default delay to mimic the average human's typing speed
```

Comment:
decided to drop the `typeDelay` here. It was added as an early attempt to deflake the tests, since sometimes characters were missed when typing. but we've since found better ways to improve stability, mostly around adding expectations to ensure the page has loaded before interacting with elements.